Add Action Property to Define Type of Plugin

aljawaid opened the pull request:
This will help identify plugins better

(The new plugin will be released soon)

aljawaid committed:
FIX: Back2TopBotm Download link

aljawaid committed:
Merge branch 'kanboard:main' into master

aljawaid committed:
Add AutomaticActionUX Plugin

aljawaid committed:
Add AutomaticActionUX Plugin

aljawaid committed:
Fix AutomaticActionUX namespace

aljawaid committed:
Add Action Plugin: KanboardEmailHistory

aljawaid committed:
Add Action Plugin: KanboardEmailHistory

Action to email the full history of a task on closure

aljawaid committed:
FIX: Missing comma and download filenames

aljawaid committed:
Merge branch 'main' into master

aljawaid committed:
Released KanboardEmailHistory v1.1

aljawaid committed:
Update plugins.json

aljawaid committed:
Update plugins.json

aljawaid committed:
Merge branch 'main' into master

aljawaid committed:
Merge branch 'kanboard:main' into master

aljawaid committed:
Add New Plugin: AgileIndicators

also correct my user handle

aljawaid committed:
Add Plugin: AgileIndicators

also fix namespace for DarkCSS
also correct my user handle

aljawaid committed:
Merge branch 'kanboard:main' into master

aljawaid committed:
New Plugin: KanboardSupport

aljawaid committed:
New Plugin: KanboardSupport

Add a support section in the Kanboard Settings interface so that end users can easily gather any information required by their internal technical support departments and for troubleshooting purposes.

aljawaid committed:
Merge branch 'kanboard:main' into master

aljawaid committed:
Update Plugin: KanboardSupport

aljawaid committed:
Merge branch 'kanboard:main' into master

aljawaid committed:
Update KanboardSupport Plugin

aljawaid committed:
Update AutomaticActionUX to v1.5

aljawaid committed:
Merge branch 'kanboard:main' into master

aljawaid committed:
Merge branch 'kanboard:main' into master

aljawaid committed:
Merge branch 'kanboard:main' into master

aljawaid committed:
Update AutomaticActionUX to v1.6

aljawaid committed:
Merge branch 'kanboard:main' into master

aljawaid committed:
Add New Plugin: Glancer

aljawaid committed:
Merge branch 'kanboard:main' into master

aljawaid committed:
Add Plugin KanboardPermalink

aljawaid committed:
Fix description for KanboardPermalink

aljawaid committed:
Fix: Change Extra JSON Property Types

- Boolean & ISO-8601 Date type

aljawaid committed:
Update README.md

- Update instructions with links to make it easier for accurate plugin submission by developers
- Add optional property types
- Specify mandatory property types
- Add code example templates

aljawaid committed:
Merge branch 'kanboard:main' into master

aljawaid committed:
NEW: Add `Last Updated` Date to Plugins

aljawaid committed:
Cleanup code

aljawaid committed:
Merge branch 'kanboard:main' into master

aljawaid committed:
Add Action Property to Define Type of Plugin

This will help identify plugins better

fguillot commented on README.md:
```diff
@@ -63,6 +64,10 @@ This website is a simple HTML static webpage to serve [Kanboard](https://kanboar
   - `true` or `false`
   - Specify whether your plugin has used any hooks
   - _Boolean type_
+  - `is_action`
```
This bullet point seems misaligned compared to others.

aljawaid replied:
done

aljawaid committed:
Fix Typo & Change Property Type

- Need to change from boolean to string because plugins can be of both types with the same plugin apparently

aljawaid committed:
Change Property Type

aljawaid committed:
Update a few property types

aljawaid committed:
NEW: Add `Last Updated` Date to Plugins

- This completes the whole directory for all plugins

fguillot committed:
Merge branch 'main' into master

fguillot commented on README.md:
```diff
@@ -23,6 +23,7 @@ This website is a simple HTML static webpage to serve [Kanboard](https://kanboar
         "has_schema": false,
         "has_overrides": false,
         "has_hooks": false,
+        "is_action": "no",
```
Why is this field not a boolean like others?

aljawaid replied:
Because we need `yes` `no' and 'both' as answers
